CHANGES.rst

```diff
@@ -5,6 +5,9 @@ CHANGES for Crate.io Documentation Theme
 Unreleased
 ----------
 
+- Add (hidden) CrateDB Tutorials project
```

Comment:
What does "hidden" mean here?

Reply:
@chaudum if you look at the templating logic, the menu item only shows up if you visit the URL for the docs project directly. which means that it won't show up as a menu item normally, and is so hidden. this is how I roll out new docs projects and get them live before they're ready to be linked to from the main site